src/sysext/sysext.c

```diff
@@ -2435,6 +2497,7 @@ static int verb_help(int argc, char **argv, void *userdata) {
                "\n%3$sOptions:%4$s\n"
                "     --mutable=yes|no|auto|import|ephemeral|ephemeral-import\n"
                "                          Specify a mutability mode of the merged hierarchy\n"
+               "                          (overrides config file)\n"
```

Comment:
Commandline options almost always override config, no need to mention this here.